shared/groupUser/group.interface.ts

```diff
@@ -0,0 +1,23 @@
+import * as mongoose from 'mongoose';
+
+export interface IPerson {
+  name: string;
+  id: string;
+  inGroups: string[];
+}

```

Comment:
import it from the person model file so it won't be repeated

Reply:
and usually, we save the members of the group in the group itself (In `usersIds` field for example), and not in the person object.

Reply:
Especially when you already have a `people` field in group, there is no need for this field (there will be duplicate data)